rakelib/artifacts.rake

```diff
@@ -59,9 +59,12 @@ namespace "artifact" do
       # plugins which use jar-dependencies.
       # See more in https://github.com/elastic/logstash/issues/4818
       "vendor/??*/**/.mvn/**/*",
+      "vendor/jruby/bin/.jruby.java_opts",
```

Comment:
Note for reviewer: without this on bundled AdoptOpenJDK the JRuby execution used to run `pleaserun` in post install script is run without the opening of modules for private introspection (miss --add-opens ...)